grading-container/script/grading_job/grading_job_builder.py

```diff
@@ -24,10 +25,10 @@ def create_grading_job(self, submission_id: int, grade_id: int, student_code: st
   def add_max_retries(self, max_retries: int):
     self.__grading_job.set_script_max_retries(max_retries)
   
-  def add_starter_code(self, starter_code: str):
```

Comment:
Is this really _add_?  Or is it _set_?  I.e. could I call these methods multiple times?